Checkers/Checkers.cs

```diff
@@ -1,86 +1,243 @@
+            for (int i = 0; i < 7; i += 2)
+            {
+                Checker c2 = new Checker("white", new int[] { 1, i });
+                Checkers.Add(c2);
+            }
+            // for the first and third lines of the black position
+            for (int i = 0; i < 7; i += 2)
+            {
+                Checker c4 = new Checker("black", new int[] { 5, i });
+                Checker c6 = new Checker("black", new int[] { 7, i });
+                Checkers.Add(c4);
+                Checkers.Add(c6);
+            }
+
+            for (int i = 1; i < 8; i += 2)
+            {
+                Checker c5 = new Checker("black", new int[] { 6, i });
+                Checkers.Add(c5);
+            }
+            // return;
         }
-        
+
         public void PlaceCheckers()
         {
             // Your code here
             return;
         }
-        
+
         public void DrawBoard()
         {
+            string[,] grid = new string[8, 8];
+            //backward slash t does create evenly tab space
+            Console.WriteLine("\t0 \t1 \t2 \t3 \t4 \t5 \t6 \t7");
+            foreach (Checker c in Checkers)
+            {
+                int x = c.Position[0];
+                int y = c.Position[1];
+                grid[x, y] = c.Symbol;
+
+            }
+            for (int i = 0; i < 8; i++)
+            {//up and down numbers from 0 to 7
+                Console.Write("{0}\t", i);
+                for (int j = 0; j < 8; j++)
+                {
+                    Console.Write("{0}\t", grid[i, j]);
+
+                }
+                Console.Write("\n");
+
+            }
             // Your code here
-            return;
+
+            // return;
         }
-        
+
         public Checker SelectChecker(int row, int column)
         {
             return Checkers.Find(x => x.Position.SequenceEqual(new List<int> { row, column }));
         }
-        
+        public void MoveChecker()
+        {
+            try
+            {
+                Console.WriteLine("Enter pickup row.");
+                int x1 = Convert.ToInt32(Console.ReadLine());
+                Console.WriteLine("Enter pickup column");
+                int y1 = Convert.ToInt32(Console.ReadLine());
+                Console.WriteLine("Enter Placement row");
+                int x2 = Convert.ToInt32(Console.ReadLine());
+                Console.WriteLine("Enter Placement column");
+                int y2 = Convert.ToInt32(Console.ReadLine());
+                Checker c1;
+                c1 = SelectChecker(x1, y1);
+                int whiteOrBlack = 0;
+                if (c1.Color == "white")
+                {
+                    whiteOrBlack = 1;
+                }
+                else if (c1.Color == "black")
+                {
+                    whiteOrBlack = -1;
+                }
+
+                if (x2 == x1 + whiteOrBlack && y2 == y1 + 1 || y2 == y1 - 1)
+
+                {
```

Comment:
Your logic for capture is a little off, you need to jump over a piece to capture it.